cluster-provision/images/vm-image-builder/fedora-sriov-lane/cloud-config

```diff
@@ -22,5 +22,8 @@ runcmd:
   - sudo dnf clean all
   - sudo hostnamectl set-hostname ""
   - sudo hostnamectl set-hostname "" --transient
-  - sudo systemctl start qemu-guest-agent
+  - sudo systemctl stop qemu-guest-agent
+  - sudo systemctl disable qemu-guest-agent
+  - sudo mv /lib/systemd/system/qemu-guest-agent.service /home/fedora/
```

Comment:
Would you try this with `disable --now` it think remove the service as well so we wont need to do it manually

Reply:
I think i tried in the past and it didn't work,
(the important thing is that it will survive reboot, and be kept offline)
please see more info on 
https://github.com/kubevirt/kubevirtci/pull/533
"due to a service dependency tree"
If needed i will retry.

Reply:
I dont recall that I had this kind of issues when creating this image and according to [systemctl docs](https://www.freedesktop.org/software/systemd/man/systemctl.html)
using `sudo systemctl disable  --now` should do the trick in one line and simply it.

Then on the "client" (kubevirt tests, cloud init) we would need to `sudo systemctl enable  --now`
and thats it, instead of messing with the service file

Reply:
I tried it in the past, and tried it again now,
it stop it not persistently between reboots.

[fedora@testvmi1 ~]$ sudo systemctl stop qemu-guest-agent
[fedora@testvmi1 ~]$ sudo systemctl disable --now qemu-guest-agent
Removed /etc/systemd/system/dev-virtio\x2dports-org.qemu.guest_agent.0.device.wants/qemu-guest-agent.service.

Reboot
and qemu-ga is back online

anyhow i am investigating what Edy asked on the other thread so in case it will work, it will replace this method.

Reply:
> You are welcome to try to login without setting it in Cloud-init UserData in the test and you will see it doesn't work.
I dont know what could-init does in the backgammon that wont let you use it w/o setting a user..
When we came up with this image one think we wanted to achieve is to not use cloud-init at all.
If you will spin up a VM with that image w/o cloud-init, you would see its there.

> The config in VM builder is not persistant.
It is resistant you could check other configurations it does e.g: remove the default host name, load kernel modules for sriov drivers..